pkg/webhook/clusterresourceplacementdisruptionbudget/clusterresourceplacementdisruptionbudget_validating_webhook_test.go

```diff
@@ -0,0 +1,315 @@
+/*
+Copyright 2025 The KubeFleet Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterresourceplacementdisruptionbudget
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+	admissionv1 "k8s.io/api/admission/v1"
+	authenticationv1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	placementv1beta1 "github.com/kubefleet-dev/kubefleet/apis/placement/v1beta1"
+	"github.com/kubefleet-dev/kubefleet/pkg/utils"
+)
+
+func TestHandle(t *testing.T) {
+	validCRPDBObject := &placementv1beta1.ClusterResourcePlacementDisruptionBudget{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test-crp",
+		},
+		Spec: placementv1beta1.PlacementDisruptionBudgetSpec{
+			MaxUnavailable: nil,
+			MinAvailable:   nil,
+		},
+	}
+	validCRPDBObjectPickNCRP := &placementv1beta1.ClusterResourcePlacementDisruptionBudget{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "crp-pickn",
+		},
+		Spec: placementv1beta1.PlacementDisruptionBudgetSpec{
+			MaxUnavailable: &intstr.IntOrString{
+				Type:   intstr.Int,
+				IntVal: 1,
+			},
+			MinAvailable: &intstr.IntOrString{
+				Type:   intstr.String,
+				StrVal: "50%",
+			},
+		},
+	}
+	invalidCRPDBObjectMinAvailablePercentage := &placementv1beta1.ClusterResourcePlacementDisruptionBudget{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test-crp",
```

Comment:
Can you change the name to pick-all-crp for the valid CRP object and use it here and other occurrences